addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -525,7 +525,7 @@ a.no-decoration {
     position: absolute !important;
     height: 150px;
     width: 150px;
-    z-index: 1020;
```

Comment:
There is probably a bootstrap variable for that :) Typically the mean between the BS element you want to be over and the BS element you want to be under.